lib/session.js

```diff
@@ -0,0 +1,47 @@
+const db = require('./db');
+const { forEachSerialP } = require('omnibelt');
+
+const createSession = async () => {
+  const beforeCommits = [];
+  const afterCommits = [];
+  const onAborts = [];
+  if (!db._client) {
+    await db.init();
+  }
+  const session = await db._client.startSession();
+
+  const abortTransaction = async (error) => {
+    await forEachSerialP(async (afterAborted) => {
```

Comment:
@spacetc62 should we use allSettled instead of forEachSerialP that way one models error wont affect the clean up of others, even though I believe these should not error.

Reply:
allSettled would run them all at once - but I think the concept is right - probably try/catch them in the forEachSerialP ?

Should you do the same for aftercommit?

Reply:
yes that's what I was thinking.